connections.js

```diff
@@ -23,6 +23,10 @@ module.exports = [
   'https://nft.mewapi.io:443',
   'https://staked.mewapi.io:443',
   'https://stakedtest.mewapi.io:443',
+  'https://c.onical.org:443', // aqua
```

Comment:
These apis aren't used anymore, please remove them except for the matomo api

Reply:
oh oops will remove on next commit